_web/LDA.rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Manova Repeated Measures"
+author: ""
+date: ""
+output:
+  html_document:
+    df_print: paged
+    toc: TRUE
+    toc_float: TRUE
+---
+
+Test Manova Repeated Measures on Dog Data
+
+#libraries
+```{r, imports1}
+library(ggplot2)
+library(reshape2)
+library(dplyr)
+library(MASS)
+```
+
+# Read data
+```{r}
+#read the input data
+bugs = read.table("../data/insect.txt", sep = " ",
+	col.names = c("bug","w.j1","w.j2","w.ae"),
+	stringsAsFactors = TRUE)
+bugs
+```
+
+```{r}
+bugs.lda = lda(formula = bug ~ w.j1 + w.j2 + w.ae, method = "moment",
+         data = bugs, 
+         prior = c(1,1)/2)
+		 
+predict(bugs.lda, data.frame(w.j1 = c(194), w.j2 = c(124), w.ae = c(49)))$posterior
```

Comment:
For all the constants, we may want to leave a comment or something to explain where they are coming from.

Reply:
I recommend you open an issue. I'm not sure what this has to do with this pull request.

Reply:
I used to my company's convention of addressing all code concerns with new code in the new PR.  But I see your point for sure.

I opened an issue for it.

Reply:
I'd definitely recommend opening an issue or a pull request of your own in this case.  Peter is generally not going to care about style issues with code he didn't write or modify.